src/main/java/spoon/reflect/visitor/ModelConsistencyChecker.java

```diff
@@ -77,11 +94,90 @@ protected void exit(CtElement e) {
 		stack.pop();
 	}
 
+	/**
+	 * Gets the list of elements that are considered inconsistent.
+	 * <p>
+	 * If {@link #fixInconsistencies} is set to true, this list will
+	 * contain all the elements that have been fixed.
+	 *
+	 * @return the invalid elements
+	 */
+	public List<InconsistentElements> inconsistentElements() {
```

Comment:
This method is only used in this class. Can it be made private? And can we use `List.copyOf()`?